### PR TITLE
[20.05] Fixes to package dependency declarations.

### PR DESCRIPTION
**Pull request by jmchilton**

### CODE REVIEW

File: packages/test_driver/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 galaxy-data
+galaxy-web-apps
 nose
 pytest
```
Comment (nsoranzo) [inline packages/test_driver/requirements.txt]:
```suggestion
pyyaml
sqlalchemy-utils
six
```

File: packages/test_driver/requirements.txt [hunk 2]
```diff
@@ -1,4 +1,5 @@
 galaxy-data
+galaxy-web-apps
```
Comment (nsoranzo) [inline packages/test_driver/requirements.txt]:
```suggestion
galaxy-app
galaxy-data
galaxy-test-base
galaxy-tool-util
galaxy-util
galaxy-web-apps
galaxy-web-framework
```

Comment (jmchilton) [reply packages/test_driver/requirements.txt]: Shouldn't we just rely on galaxy-web-apps to recursively bring those in?

Comment (nsoranzo) [reply packages/test_driver/requirements.txt]: I think the correct approach is to add all packages from which this package directly imports a module, in the same way we list both bioblend and requests in `lib/galaxy/dependencies/pipfiles/default/Pipfile` even if the former depends on the latter.

Comment (jmchilton) [reply packages/test_driver/requirements.txt]: I'm skeptical it is appropriate here... but I do see the value generally. I've merged this change - thanks so much and thanks for the explanation.

**Review by nsoranzo — APPROVED** [review]

**Comment by jmchilton** [comment]
Thanks for the merge @nsoranzo !

**Pull request merged**